src/utils/useFetchPeople.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const useFetchPeople = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setErrorMessage('Something went wrong');
```

Comment:
enum

Reply:
All done, many thanks for your suggestions! 🙌